app/controllers/concerns/sign_in/authentication.rb

```diff
@@ -36,5 +40,16 @@ def authenticate_access_token
     def load_user
       SignIn::UserLoader.new(access_token: @access_token).perform
     end
+
+    def handle_authenticate_error(error, status: :unauthorized, level: :error)
```

Comment:
Looks like we aren't ever changing `level`, we could prob leave out the param and just code it to `:error` in the `log_message_to_sentry` call. Also if we don't log when access_token_expired_error occurs we could prob hard code `status` as well